docker-compose.yml

```diff
@@ -25,11 +25,12 @@ services:
       - spid-testenv2
 
   spid-testenv2:
-    image: italia/spid-testenv2:latest
+    image: italia/spid-testenv2:1.1.0
```

Comment:
Why stick to a specific version and not use the latest available?

Reply:
It's good practice to stick to a specific version for repeatability. A new major release could bring changes that break our assumptions on how the testenv behaves, such as an incompatible configuration format.